src/organisms/announcementCard/AnnouncementCard.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from 'react';
+import { Text } from '../../atoms/texts/text';
+import {
+  AnnouncementCardContainer, AnnouncementText, Content, Cover,
+} from './styles';
+import { AnnouncementCardProps } from './types';
+
+export const AnnouncementCard: FC<AnnouncementCardProps> = ({
+  coverUrl, releaseDate, title, purifiedContent, width,
+}) => (
+  <AnnouncementCardContainer width={width}>
+    <Cover coverUrl={coverUrl} />
+    <Content>
```

Comment:
@ArinCantCode no spacing into this component ? ask Michal to review this component please.